test/parallel/test-debug-port-from-cmdline.js

```diff
@@ -38,11 +38,12 @@ function processStderrLine(line) {
 
 function assertOutputLines() {
   const expectedLines = [
-    'Starting debugger agent.',
-    'Debugger listening on 127.0.0.1:' + debugPort,
+    /Starting debugger agent./,
```

Comment:
Probably a good idea to start with `^` and end with `$` for this and the other regular expressions.